etc/kayobe/pulp.yml

```diff
@@ -369,6 +369,16 @@ stackhpc_pulp_repository_rpm_repos: >-
   {%- endfor -%}
   {{ rpm_repos }}
 
+# Mellanox OFED version
+stackhpc_pulp_mlnx_ofed_version: 24.04-0.7.0.0
+
+# Mellanox OFED repositories
+stackhpc_pulp_repo_mlnx_ofed:
+  name: Mellanox Technologies mlnx_ofed {{ stackhpc_pulp_mlnx_ofed_version }}
```

Comment:
Needs to include the Rocky minor version in the name:

> Use one Pulp RPM repo for each version of OFED and Rocky minor version

Reply:
Added Rocky9.x to repo name.

Reply:
Has this been reverted @assumptionsandg still looks the same to me